waiter/src/waiter/settings.clj

```diff
@@ -134,7 +137,9 @@
     (if (.exists config-file)
       (do
         (log/info "reading settings from file:" config-file-path)
-        (let [edn-readers {:readers {'regex (fn [expr] (re-pattern expr))}}
+        (let [edn-readers {:readers {'regex (fn [expr] (re-pattern expr))
+                                     'config/env #(env %)
```

Comment:
Cook uses `config/env` and `config/env-int` -- should we instead switch to `config/regex`?